packages/paste-core/components/data-grid/__tests__/index.spec.tsx

```diff
@@ -231,3 +232,43 @@ describe('Data Grid', () => {
     });
   });
 });
+
+describe('i18n', () => {
+  describe('DataGridHeaderSort', () => {
+    it('should have default ascending label', () => {
+      render(<DataGridHeaderSort direction="ascending" />);
+      const button = screen.getByRole('button', {name: 'Sort ascending'});
+      expect(button).toBeDefined();
+    });
+
+    it('should have default descending label', () => {
+      render(<DataGridHeaderSort direction="descending" />);
+      const button = screen.getByRole('button', {name: 'Sort descending'});
+      expect(button).toBeDefined();
+    });
+
+    it('should have default unsorted label', () => {
+      render(<DataGridHeaderSort direction="none" />);
+      const button = screen.getByRole('button', {name: 'Unsorted'});
+      expect(button).toBeDefined();
+    });
+
+    it('should use i18nAscendingLabel for the ascending label', () => {
+      render(<DataGridHeaderSort direction="ascending" i18nAscendingLabel="Tri croissant" />);
+      const button = screen.getByRole('button', {name: 'Tri croissant'});
```

Comment:
🥐

Reply:
I KNOW! I forgot that its an actual word. I did so much research on this.